hexrd/ui/material_structure_editor.py

```diff
@@ -304,7 +316,7 @@ def coords_equal(v1, v2, tol=1.e-5):
 
             U_array = new_U_array
 
-        for i, atom in enumerate(info_array):
+        for i, atom in enumerate(pos_array):
```

Comment:
I'm confused what Line 319-350 is for? Is it checking if two sites have the same atoms?

Reply:
It is actually checking if two atoms have a matching site. This is for generating the GUI based off of what is in the `Material` class. It is because the GUI was designed to have the user select a site and then view/edit atoms in that particular site. So it compares the coordinates to group atoms into different sites.